assets/js/components/Dashboard/PendingScreensPage.tsx

```diff
@@ -36,15 +36,15 @@ const PendingScreensPage: ComponentType = () => {
 
   const fetchData = useCallback(() => {
     fetchExistingScreensAtPlacesWithPendingScreens().then(
-      ({ places_and_screens, version_id, last_modified_ms }) => {
+      ({ places_and_screens, eTag, last_modified_ms }) => {
```

Comment:
Should `version_id` be changed to `eTag`? We shouldn't be using etags anymore now that we can look at object versions so the `fetch_config` returns what is called a `version_id`.

Reply:
Oof my bad! I thought we were still using ETags.

This is sort of an odd situation since it's essentially serving the role of an ETag in the context of communication between the screenplay client and server.

I'm trying to follow HTTP standards in terms of header names/status codes for this stuff, but maybe that's silly. Will update relevant code to something more sensible.

Reply:
Ok I thought about this some more—since the main issue is that there was confusion around using S3 version ID as ETag in the context of our own server's API, I figured the proper solution was... to just start using the S3 ETag all the way through!

So now, the client gets both a version ID and an ETag.
It uses the ETag for ETag-y things (avoiding accidental overwrites), and the version ID for versioning things (reverting to a previous config version, when necessary)